test/Abp.Zero.SampleApp.Tests/EntityHistory/SimpleEntityHistory_Test.cs

```diff
@@ -866,46 +866,39 @@ public void Should_Not_Write_History_If_Invalid_Entity_Has_Property_With_Audited
         }
 
         [Fact]
-        public void Should_Not_Write_History_For_Not_Audited_Entities_Shadow_Property()
+        public void Should_Not_Write_History_For_Audited_Entity_By_Default()
         {
-            Resolve<IEntityHistoryConfiguration>().IsEnabled = true;
-            Resolve<IEntityHistoryConfiguration>().Selectors.Clear();
-
-            _entityHistoryStore.ClearReceivedCalls();
-            //Arrange
```

Comment:
Please restore these comments.

Reply:
He already answer that to me :) https://github.com/aspnetboilerplate/aspnetboilerplate/pull/5151#discussion_r375677064

Reply:
I mean the `//Arrange` comment, since we have `//Assert` 🙊

Reply:
Done